build.gradle

```diff
@@ -65,6 +65,7 @@ dependencies {
       compile 'commons-cli:commons-cli:1.3.1'
       compile 'xpp3:xpp3:1.1.4c'
       compile 'joda-time:joda-time:2.9.3'
+      compile 'com.carrotsearch:hppc:0.7.2'
```

Comment:
the `IntCollection` and `IntCursor` classes used by the sqlList functions require this dependency. Was already in `commcare-android`